frontend/src/components/Footer.vue

```diff
@@ -3,18 +3,18 @@
     <v-card id="tiers__sponsors" width="100vw" class="white--text text-center">
       <v-card-text class="white--text">
         <section class="tiers__grid">
-          <a v-for="sponsor in tierOne" class="tiers__box" :href="sponsor.link" :key="sponsor.id">
-            <img :src="`data:image/png;base64,${sponsor.logo}`" style=" max-width:300px;max-height:105px;" />
+          <a v-for="sponsor in tierOne" class="tiers__box" :href="sponsor.url" :key="sponsor.id">
+            <img :src="sponsor.logo" style=" max-width:300px;max-height:105px;" />
           </a>
         </section>
         <section class="tiers__grid">
-          <a v-for="sponsor in tierTwo" class="tiers__box" :href="sponsor.link" :key="sponsor.id">
-            <img :src="`data:image/png;base64,${sponsor.logo}`" style="max-width:200px;max-height:75px" />
+          <a v-for="sponsor in tierTwo" class="tiers__box" :href="sponsor.url" :key="sponsor.id">
+            <img :src="sponsor.logo" style="max-width:200px;max-height:75px" />
           </a>
         </section>
         <section class="tiers__grid">
-          <a v-for="sponsor in tierThree" class="tiers__box" :href="sponsor.link" :key="sponsor.id">
-            <img :src="`data:image/png;base64,${sponsor.logo}`" style="max-width:100px;max-height:50px" />
+          <a v-for="sponsor in tierThree" class="tiers__box" :href="sponsor.url" :key="sponsor.id">
+            <img :src="sponsor.logo" style="max-width:100px;max-height:50px" />
```

Comment:
The base64 images are not being loaded. Make sure you use the implementation from the dev branch with the "data:image/png;" prefix.

Reply:
Fixed! Thanks!! forgot bout it